.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+
+references:
+  cache_key: &cache_key
+    key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+
+jobs:
+  build:
+    docker:
+      - image: circleci/android:api-27-alpha
+    environment:
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+    steps:
+      - checkout
+      - restore_cache:
+          <<: *cache_key
+      - run:
+          name: Download Dependencies
+          command: ./gradlew androidDependencies
+      - save_cache:
+          <<: *cache_key
+          paths:
+            - ~/.gradle
+      - run:
+          name: Run JVM Tests & Lint
+          command: ./gradlew check
+      - store_artifacts:
+          path: app/build/reports
```

Comment:
Do you not want to store the APKs?

Reply:
Not at this point - just build and test for now.